bindings/go/evmc/evmc.go

```diff
@@ -391,6 +394,12 @@ func (vm *VMSteppable) StepN(params StepParameters) (res StepResult, err error)
 		stack = (*C.evmc_uint256be)(unsafe.Pointer(&params.Stack[0]))
 	}
 
+	pnr := new(runtime.Pinner)
```

Comment:
The pinning of the blob hash array should also be done in the `Execute` function above, to get support for blob hashes also in the regular case, not running the StepN function.